go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/withastro/compiler
 
-go 1.19
+go 1.20
```

Comment:
```suggestion
go 1.22
```

Reply:
`go.mod` is just the minimum version

Reply:
Why do we require a minimum version, but then build the program in a more recent version? I don't understand

Reply:
If a contributor has 1.20 installed, it would work the same as 1.22. It's not necessary to make them upgrade their toolchain.

Reply:
What's the minimum go version needed by the testing library you wanted to use?

Reply:
This left me with more doubts and questions:
- we test and **compile** against a different target. What could work locally might not work on CI and vice versa
- we don't release go libs, why do we require different versions?
- At the moment we all point to 1.19, but now we are branching. Does the contribution guide explain this?

Reply:
Can you answer my questions?

The lib that I need requires 1.21